package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "webpack --config webpack.prod.config.js && babel server -d server-dist",
     "start:dev": "nodemon --exec NODE_ENV=development babel-node server/index.js",
-    "test": "jest",
+    "test": "jest --watch",
```

Comment:
Please remove `--watch` flag as it breaks CI/CD. If you want to use watch, just do `npx jest --watch` from CLI